### DISCUSSION TITLE
Example of a Boltzmann Selection Function

### DISCUSSION THREAD

**Discussion by chadb56** [ahmedfgad/GeneticAlgorithmPython#273] [Q&A]
Can you provide me with an example of how to implement a selection function that selects the parents from a Boltzmann distribution?  I was reading [this](https://towardsdatascience.com/evolving-a-neural-network-in-a-sparse-reward-environment-db1f596d5dc6) article in which they use a Boltzmann selection method, and I would like to replicate their results using PyGad, but I'm not sure how to properly implement.

**Comment by ahmedfgad** [comment]
There is no example yet about selecting parents using Boltzmann distribution.

But you can create a custom parent selection function/method and assign it to the `parent_selection_type` parameter. This is a dummy example: https://github.com/ahmedfgad/GeneticAlgorithmPython/blob/98b187ba79849930887a0c0a4d11f9cf7780ee3d/examples/example_custom_operators.py#L23
